bitnami/kube-state-metrics/templates/servicemonitor.yaml

```diff
@@ -24,6 +24,12 @@ spec:
       {{- if .Values.serviceMonitor.scrapeTimeout }}
       scrapeTimeout: {{ .Values.serviceMonitor.scrapeTimeout }}
       {{- end }}
+      {{- if .Values.serviceMonitor.honorLabels }}
+      honorLabels: true
+      {{- end }}
```

Comment:
If this is a boolean value can this just be:
```
      honorLabels: {{ .Values.serviceMonitor.honorLabels }}
```

Reply:
It could, but this would lead to the `honorLabels` key being `null` if `serviceMonitor.honorLabels` is not defined.
I've pushed changes to address that: now `honorLabels` is omitted unless it has been explicitly set in the chart values.